e2e/portalicious/tests/DoPayment/ShowPaymentSummary.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { test } from '@playwright/test';
+
+import { SeedScript } from '@121-service/src/scripts/seed-script.enum';
+import NLRCProgram from '@121-service/src/seed-data/program/program-nlrc-ocw.json';
+import { seedIncludedRegistrations } from '@121-service/test/helpers/registration.helper';
+import { resetDB } from '@121-service/test/helpers/utility.helper';
+import { getAccessToken } from '@121-service/test/helpers/utility.helper';
+import { registrationsOCW } from '@121-service/test/registrations/pagination/pagination-data';
+
+import BasePage from '@121-e2e/portalicious/pages/BasePage';
+import LoginPage from '@121-e2e/portalicious/pages/LoginPage';
+import PaymentsPage from '@121-e2e/portalicious/pages/PaymentsPage';
+
+test.beforeEach(async ({ page }) => {
+  await resetDB(SeedScript.nlrcMultiple);
+  const programIdOCW = 3;
+  const OcwProgramId = programIdOCW;
+
+  const accessToken = await getAccessToken();
+  await seedIncludedRegistrations(registrationsOCW, OcwProgramId, accessToken);
+
+  // Login
+  const loginPage = new LoginPage(page);
+  await page.goto('/');
+  await loginPage.login(
+    process.env.USERCONFIG_121_SERVICE_EMAIL_ADMIN,
+    process.env.USERCONFIG_121_SERVICE_PASSWORD_ADMIN,
+  );
+});
+
+test('[31971] Show payment summary', async ({ page }) => {
+  const basePage = new BasePage(page);
+  const payments = new PaymentsPage(page);
```

Comment:
```suggestion
  const paymentsPage = new PaymentsPage(page);
```

Not sure what you usually do now, but this is slightly more clear to me.

Reply:
I'll keep it as you suggested